tests/jenkins/TestCopyContainer.groovy

```diff
@@ -38,28 +39,34 @@ class TestCopyContainer extends BuildPipelineTest {
     @Test
     public void testCopyContainerDockerStagingToDockerProd_verifyShellCommand() {
         super.testPipeline("tests/jenkins/jobs/DockerCopy_Jenkinsfile")
-         def shellCommands = getCommandExecutions('sh', 'gcrane').findAll {
-            shCommand -> shCommand.contains('gcrane')
-         }
 
-         assertThat(shellCommands.size(), equalTo(1))
-         assertThat(shellCommands, hasItem("gcrane cp opensearchstaging/alpine:3.15.4 opensearchproject/alpine:3.15.4; docker logout".toString()))
+        String crane_str = 'set -x && crane cp opensearchstaging/alpine:3.15.4 opensearchproject/alpine:3.15.4'
+        assertThat(getShellCommands('sh', 'crane'), hasItem(crane_str))
+
+        String docker_str = 'set +x && docker logout && docker logout opensearchproject'
```

Comment:
Didn't get why we are logging out both?

Reply:
Because this `craneCopy` function is used for not only docker hub but also ecr, logging out both is not going to send error and we make sure we are logging out of either.